### DISCUSSION TITLE
Weather modification in battle

### DISCUSSION THREAD

**Discussion by Thyphon76** [pokemon-tabletop-reunited/ptr1e#764] [Q&A] [answered]
Hello,
I'm stuck with some weather abilities effect. I didn't know how to fix it: For exemple, when I try to use Sunny Day, the game tell me "It appears this moves dosen't have any special effect" like on the screen. I didn't understand why it's happen.. Someone can help me please?
![Capture d'écran 2024-05-09 221144](https://github.com/dylanpiera/Foundry-Pokemon-Tabletop-United-System/assets/169399690/558510b9-bca4-46b0-8ef1-e9fc7d5a3bf7)

**Comment by dylanpiera** [comment, marked as answer]
Hi @Thyphon76, sorry usually we get questions on our Discord so I missed this one!

Status moves can have effects applied on them, these will apply to the targets of said status moves.
Weather moves are global effects and therefore do not have this functionality.

Instead you can use the 'Global Effects' over on the left hand toolbar to apply the appropriate weather effect.

**Reply by Thyphon76** [reply]
ok I see, I will try thank you
